include/xml.php

```diff
@@ -44,6 +47,15 @@ public static function from_array($array, &$xml, $remove_header = false, $namesp
 		}
 
 		foreach($array as $key => $value) {
+			if (!isset($element) AND isset($xml))
+				$element = $xml;
+
+			if (is_integer($key)) {
+				if (isset($element))
+					$element[0] = $value;
```

Comment:
for this I get the error `PHP Warning: It is not yet possible to assign complex types to properties in /.../include/xml.php on line 55` with PHP 7.0.9.

Posting new entries works (but raises the error) fetching the home time line does not (using Friendly).